plugins/brew.ts

```diff
@@ -0,0 +1,62 @@
+import { spawn } from "../src/helpers.ts";
+import Plugin from "../src/plugin.ts";
+
+class Brew implements Plugin {
+  name = "brew";
+  commands: Record<string, (params: string[]) => Promise<void>>;
+  constructor() {
+    this.commands = {
+      search: (args: string[]) => spawn(this.name, ["search", ...args]),
+      info: (args: string[]) => spawn(this.name, ["info", ...args]),
+      install: (args: string[]) => spawn(this.name, ["install", ...args]),
+      update: (args: string[]) => spawn(this.name, ["update", ...args]),
+      upgrade: (args: string[]) => spawn(this.name, ["upgrade", ...args]),
+      uninstall: (args: string[]) => spawn(this.name, ["uninstall", ...args]),
+      list: (args: string[]) => spawn(this.name, ["list", ...args]),
+      config: (args: string[]) => spawn(this.name, ["config", ...args]),
+      doctor: (args: string[]) => spawn(this.name, ["doctor", ...args]),
+      create: (args: string[]) => spawn(this.name, ["create", ...args]),
+      edit: (args: string[]) => spawn(this.name, ["edit", ...args]),
+      help: () => {
+        console.log(`
+        Example usage:
+            brew search TEXT|/REGEX/
```

Comment:
You can directly invoke `search` command inside the REPL:  replace all `brew` references `brew search ` -> `search`